### PR TITLE
Add footer component

### PR DESCRIPTION
**Pull request by nabaroa**
# v5.17.0

### Working on production: https://transferwise.com/gb/compare/

I changed de footer component styles, because the way of styling the `a` element with two pseudo clases (`.footer a:not(.btn):not(.tw-select-option-link)`) as it is currently, was giving specificity issues and a complex inheritance. To solve this, I avoid styling the `a` element by default and created clases to add to the title and the links.

- `.footer-title` to block titles.
- `.footer-link` to links.

I added `.tiny` utility class for very small copies, as the copyright.

I extracted the footer styles from `transferwise.less` and created a proper partial to the footer.

### Default footer
image

### Inverse footer
image

### CODE REVIEW

File: less/footer.less [hunk 1]
```diff
@@ -0,0 +1,70 @@
+.footer {
+  background-color: @gray-base;
+  font-size: @font-size-small;
+  line-height: @line-height-computed;
+
+  .footer-link,
```
Comment (matallo) [inline less/footer.less:6]: having their own class now, do we really need them to be nested? I suppose it's due to some specificity issue

**Comment by matallo** [comment]
> To solve this, I avoid styling the a element by default and created clases to add to the title and the links.

😍 

> I extracted the footer styles from transferwise.less and created a proper partial to the footer.

😍 😍 

did we finally ditched the possibility of having flexbox grids?

File: less/footer.less (hunk 1 above)
Comment (nabaroa) [reply less/footer.less:6]: Yeap, that's the reason :)

**Comment by nabaroa** [comment]
We're taking into account the possibility of adding flexbox to the system. The footer would be a layout that could be benefited with it.

**Review by matallo — APPROVED** [review]
👍

**Pull request merged**